.idea/Data-Structures-Algorithms-DSA-.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
And this? actually  have never worked with xml files so no idea...

Reply:
To be honest I have no idea. I just worked on those python files and just committed them to my fork.

Reply:
Don't merge them or add them to gitignore. Maybe it's a pycharm thing.

Reply:
Yeah I looked it up it's a pycharm thing oof.

Reply:
ok got it...